docs/changelog.rst

```diff
@@ -8,6 +8,7 @@ Development
 ===========
 - (Fill this out as you fix issues and develop your features).
 - Fix for uuidRepresentation not read when provided in URI #2741
+- Add option to user array_filters https://www.mongodb.com/docs/manual/reference/operator/update/positional-filtered/
```

Comment:
Add the `#2769` at the end to keep it consistent with the other logs (PR number if no associated issue number)